coral-component-calendar/src/scripts/Calendar.js

```diff
@@ -28,9 +28,11 @@ function isDateInRange(date, startDate, endDate) {
 
   if (startDate === null && endDate === null) {
     return true;
-  } else if (startDate === null) {
+  }
```

Comment:
"1tbs" braces-style appears more readability(my-thought).

Reply:
ESLint throws warnings in this project:

```
/github.com/adobe/coral-spectrum/coral-component-calendar/src/scripts/Calendar.js
  353:5   warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  373:5   warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  585:23  warning  Method '_setActiveDescendant' has a complexity of 17. Maximum allowed is 15  complexity
  601:13  warning  Blocks are nested too deeply (5). Maximum allowed is 4                       max-depth
  603:13  warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  607:9   warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  639:7   warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  729:13  warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  732:11  warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  815:9   warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  862:5   warning  Closing curly brace appears on the same line as the subsequent block         brace-style
  879:5   warning  Closing curly brace appears on the same line as the subsequent block         brace-style 
 ```

Reply:
@majornista 
This maybe due to https://github.com/adobe/coral-spectrum/blob/master/coral-gulp/configs/eslint.conf.js#L8 . This review comment was my personal opinion. I found "1tbs" more readable.  Feel free to resolve it, as per your convenience.

Reply:
I think following the preferred convention of coral-spectrum would be better.